tests/dummy/app/controllers/browsers.js

```diff
@@ -69,9 +63,6 @@ export default Ember.Controller.extend({
             name: 'span',
             lib: { 'sec': true }
         }, {
-            name: 'tab-pane',
```

Comment:
This is not related to the `sl-panel` component.  Is this modification intentional?